test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -37,7 +37,6 @@ const ignoredMocks = [
   'error-okta-verify-totp.json',
   'error-internal-server-error.json',
   'error-identify-access-denied.json',
-  'error-google-authenticator-otp.json',
```

Comment:
This file has been removed when I was working on softtoken i18n, so removing it from the filter.